src/App.js

```diff
@@ -1,28 +1,42 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
-import { BrowserRouter } from 'react-router-dom';
+import { 
+  BrowserRouter as Router,
+  Route,
+  Link
+} from 'react-router-dom';
+import OpenTrunk from "./open-trunk";
+import Books from "./books";
+import Equipment from './equipment';
+import MagicalBeasts from './magical-beasts';
+import Potions from './potions';
 
-class App extends Component {
+class ClosedTrunk extends Component {
   render() {
     return (
       <div className="App">
         <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
+          <h1>Magic Trunk</h1>
+          <Link to="/open-trunk">Open The Trunk</Link>
         </header>
-        <BrowserRouter></BrowserRouter>
       </div>
+    )
+  }
+}
+
+class App extends Component {
+  render() {
+    return (
+      <Router>
+        <div className="main-container">
+          <Route path="/" exact component={ClosedTrunk}></Route>
+          <Route path="/open-trunk" exact component={OpenTrunk}></Route>
+          <Route path="/open-trunk/books" exact component={Books}></Route>
+          <Route path="/open-trunk/equipment" exact component={Equipment}></Route>
+          <Route path="/open-trunk/magical-beasts" exact component={MagicalBeasts}></Route>
+          <Route path="/open-trunk/potions" exact component={Potions}></Route>
```

Comment:
Can we pull out the routes and components into an object, and loop over it?

Reply:
Done.